packages/web/pages/settings/feeds/index.tsx

```diff
@@ -229,10 +229,17 @@ export default function Rss(): JSX.Element {
                   }}
                 >
                   <SpanBox>{`URL: ${subscription.url}`}</SpanBox>
-                  <SpanBox>{`Last refreshed: ${
-                    subscription.lastFetchedAt
-                      ? formattedDateTime(subscription.lastFetchedAt)
-                      : 'Never'
+                  {/* show failed timestamp instead of last refreshed timestamp if the feed failed to refresh */}
+                  <SpanBox>{`${
```

Comment:
one small tweak I'd make here is we could set the color to read if failedAt is displayed. Probably something like this:

```
<SpanBox css={{ color: failedAt ? 'red': undefined }}>
```

Reply:
<img width="361" alt="Screenshot 2024-04-08 at 4 32 49 PM" src="https://github.com/omnivore-app/omnivore/assets/12963965/dbce78c7-3654-4c54-8e40-64d35cc44532">

Looks good to me

Reply:
Comparison:

<img width="299" alt="Screenshot 2024-04-08 at 4 33 42 PM" src="https://github.com/omnivore-app/omnivore/assets/12963965/fd7471bd-9a6d-4d75-850b-76b9a7d6d4c6">